util/build_configs/cray-internal/generate-modulefile.bash

```diff
@@ -135,8 +132,9 @@ if { [string match aarch64 $CHPL_HOST_ARCH] } {
 }
 
 if { ! [ info exists env(PE_ENV) ] } {
-    module load PrgEnv-gnu
-    set compiler GNU
+    puts stderr "Error: The Chapel module requires a PrgEnv environment"
+    puts stderr "to be loaded."
+    exit 1
```

Comment:
This is the only change I'm not 100% on board with. I kinda get the need, but it seems unfortunate to require users to have to manually load a module before ours.